src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/Microsoft.AspNetCore.Razor.VSCode.Extension.npmproj

```diff
@@ -7,19 +7,6 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
```

Comment:
I commented on the PR itself, but just so its not missed, if we still have projects that need `node` etc. then I think we are better off keeping the TextMate tests, until they can all move together

Reply:
Thanks for the feedback. I've added back the project for now and will look into moving all the grammar code in a follow up PR.